Change list table to use virutal module instaed of schema (#63)
datajoint/pharus: CHANGELOG.md

Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.

> ## [Unreleased]

Reviewer: Looks like you missed adding the link down at the bottom for `Unreleased`. Could you update it similarly to https://github.com/datajoint/pharus/commit/e6ae1bfe96562571eafc3ca9d9705c0d8a43d7af?

### Fixed

> - Fixed list_table create schema if user gives an nonexistent schema as params (#65) PR #63

Reviewer: grammar issues
```suggestion
- Fixed `list_table` creating a schema if user gives a nonexistent schema passed as a query parameter. (#65) PR #63
```

Author: I reword this one to be  a bit more clear
